src/ExpressionEvaluator/Core/Test/FunctionResolver/VisualBasicFunctionResolverTests.cs

```diff
@@ -0,0 +1,147 @@
+                VerifySignatures(requestCSharp);
+                resolver.EnableResolution(process, requestVB);
+                VerifySignatures(requestVB, "C.F()");
+                resolver.EnableResolution(process, requestCPP);
+                VerifySignatures(requestCPP);
+            }
+
+            // Module loaded after requests.
+            module = new Module(bytes);
+            using (var process = new Process())
+            {
+                var requestDefaultId = new Request(null, MemberSignatureParser.Parse("F"), Guid.Empty);
+                var requestMethodId = new Request(null, MemberSignatureParser.Parse("F"), DkmLanguageId.MethodId);
+                var requestCSharp = new Request(null, MemberSignatureParser.Parse("F"), csharpLanguageId);
+                var requestVB = new Request(null, MemberSignatureParser.Parse("F"), vbLanguageId);
+                var requestCPP = new Request(null, MemberSignatureParser.Parse("F"), cppLanguageId);
+                resolver.EnableResolution(process, requestCPP);
+                resolver.EnableResolution(process, requestVB);
+                resolver.EnableResolution(process, requestCSharp);
+                resolver.EnableResolution(process, requestMethodId);
+                resolver.EnableResolution(process, requestDefaultId);
+                process.AddModule(module);
+                resolver.OnModuleLoad(process, module);
+                VerifySignatures(requestDefaultId, "C.F()");
+                VerifySignatures(requestMethodId);
+                VerifySignatures(requestCSharp);
+                VerifySignatures(requestVB, "C.F()");
+                VerifySignatures(requestCPP);
+            }
+        }
+
+        [Fact]
+        public void DifferentCase_MethodsAndProperties()
+        {
+            var source =
+@"class A
+{
+    static void method() { }
+    static void Method(object o) { }
+    object property => null;
+}
+class B
+{
+    static void Method() { }
+    object Property { get; set; }
+}";
+            var compilation = CreateCompilationWithMscorlib(source);
+            using (var process = new Process(new Module(compilation.EmitToArray())))
+            {
+                var resolver = Resolver.VisualBasicResolver;
+                Resolve(process, resolver, "method", "A.method()", "A.Method(System.Object)", "B.Method()");
+                Resolve(process, resolver, "Method", "A.method()", "A.Method(System.Object)", "B.Method()");
+                Resolve(process, resolver, "[property]", "A.get_property()", "B.get_Property()", "B.set_Property(System.Object)");
+                Resolve(process, resolver, "[Property]", "A.get_property()", "B.get_Property()", "B.set_Property(System.Object)");
+                Resolve(process, resolver, "[PROPERTY]", "A.get_property()", "B.get_Property()", "B.set_Property(System.Object)");
+                Resolve(process, resolver, "GET_PROPERTY", "A.get_property()", "B.get_Property()");
```

Comment:
Ported the `Properties` test to `VisualBasicFunctionResolverTests` which includes `P()` and `P(Object)`.